renderer/utils/reports.ts

```diff
@@ -80,7 +80,7 @@ export function parseReport(fileContent: string) {
       // should skip registraion when task starts from !
       const isBreak = workingTimeRegex.test(currentLine) || !currentLine;
       if (isBreak) {
-        registration.project = currentLine;
+        registration.project = !currentLine ? "!" : currentLine;
```

Comment:
Approve this solution with alex pls. He spoke about just highlight such cases. Also we have "!" and the end of the day, we speak about hide it in this cases.